### ISSUE TITLE
Put up a demo / video somewhere of the component in action

### ISSUE THREAD

**Issue by d33tah** [django-money/django-money#336]
Interesting project! I'm tempted to try it out and I'm pretty sure that people would be quicker to adopt it if they could see the front-end features live - either as a video or an actual page. What do you think about that?

**Comment by benjaoming** [comment]
I think that would be over-doing it and a bit of a self-contradiction:

If django-money is so complicated that it cannot be explained by a code example and a screenshot, but it needs a video, then it's broken by design IMO.

**Comment by d33tah** [comment]
But there's not even a screenshot exposed anywhere.

**Comment by benjaoming** [comment]
Nah true, but I think the stars of the repo would convince someone that this is useful, rather than a screenshot :)

**Comment by d33tah** [comment]
@benjaoming Here's my perspective: it's not about whether it's useful at all, but whether it's useful for one's particular use case. This is when it is helpful to have a demo / video: when you're exploring lots of projects and looking for one with particular looks/feature set, basically one that fits your project the most, it's good if the maintainer puts some teaser on the website. Also, lots of people are browsing sites like hacker news and stumbling upon this kind of projects and many times when you stumble upon it randomly, it's easier to remember a screenshot than a name - it's just another way of amplifying memory and giving enough information to subconsciousness to have it work for you as well.

Also, if you put it somewhere high in docs/readme, it personally gives me the "this is easy" vibe - I just saw the screenshot, maybe also a small snippet that shows how easy it is to integrate the thing, it makes it more likely that I come back and choose the project as the first one to try even if it didn't have most stars in its category.

Here's another reason: at one point you might just decide not to maintain the project anymore because of some unforeseeable reasons, perhaps even without marking the project as "not in development anymore". From this point on, I'd expect any project to start to decay, losing compatibility with whatever it's attached to and researching "what does it do exactly" is more complicated because you might need to port it or solve some installation problems.

I just gave you quite a couple of arguments for adding at least a screenshot. Any chance any one of them convinces you?
